package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "foundryvtt-mindflayer",
+  "version": "1.5.1-development",
```

```suggestion
  "version": "2.0.0-dev",
```

we could change this to 2.0.0-dev(elopment) since this will be a major release

---

This should be done via the conventional commit automatically